runtime/tests/interpreter/reference_test.go

```diff
@@ -472,23 +472,17 @@ func TestInterpretResourceReferenceAfterMove(t *testing.T) {
 			address,
 		)
 
-		arrayRef := &interpreter.EphemeralReferenceValue{
-			Authorized: false,
-			Value:      array,
-			BorrowedType: &sema.VariableSizedType{
+		arrayRef := interpreter.NewUnmeteredEphemeralReferenceValue(
+			false,
+			array,
+			&sema.VariableSizedType{
```

Comment:
maybe keep the labels here, although it's just a test case
```suggestion
		arrayRef := interpreter.NewUnmeteredEphemeralReferenceValue(
			Authorized:   false,
			Value:        array,
			BorrowedType: &sema.VariableSizedType{
```

Reply:
This was changed from the struct-literal to a function call (a constructor). So the labels are gone now.